src/sentry/integrations/slack/notifications.py

```diff
@@ -142,6 +143,58 @@ def get_channel_and_token_by_recipient(
     return output
 
 
+def _notify_recipient(
+    notification: BaseNotification,
+    recipient: Team | User,
+    attachments: SlackBody,
+    channel: str,
+    token: str,
+    is_multiple: bool,
+) -> None:
+    # unfurl_links and unfurl_media are needed to preserve the intended message format
+    # and prevent the app from replying with help text to the unfurl
+    payload = {
+        "token": token,
+        "channel": channel,
+        "link_names": 1,
+        "unfurl_links": False,
+        "unfurl_media": False,
+        "text": notification.get_notification_title(),
+        "attachments": json.dumps(attachments),
+    }
+    log_params = {
+        "notification": notification,
+        "recipient": recipient.id,
+        "channel_id": channel,
+        "is_multiple": is_multiple,
+    }
+    post_message.apply_async(
+        kwargs={
+            "payload": payload,
+            "log_error_message": "notification.fail.slack_post",
+            "log_params": log_params,
+        }
+    )
+    notification.record_notification_sent(recipient, ExternalProviders.SLACK)
+
+
+def check_if_is_multiple(
+    notification: BaseNotification,
+    recipient: Team | User,
+    tokens_by_channel: Mapping[str, str],
+) -> bool:
+    is_multiple = len(tokens_by_channel.keys()) > 1
+    if is_multiple:
```

Comment:
@mgaeta wondering why we have all this logic for `is_multiple`. Is this doing anything functional? Or is it just sued for logging?

Reply:
Yes `is_multiple` is just for logging.